catroid/src/main/java/org/catrobat/catroid/sensing/CollisionInformation.java

```diff
@@ -167,7 +167,7 @@ private void createCollisionPolygon(String path) {
 				if (pointToPointDistance(simplified.get(0), simplified.get(simplified.size() - 1)) < epsilon) {
 					simplified.remove(simplified.size() - 1);
 				}
-
+				
```

Comment:
Why space?

Reply:
After I moved the functions it added space, will fix it now.